docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1269 @@
+        which clearly describes it, such as <emphasis>thor_50</emphasis> as in
+        the following example.</para>
+
+        <para><programlisting>-name: thor_50</programlisting></para>
+
+        <para>Updating the <emphasis>numWorkers</emphasis> value will restart
+        the Thor agent listening to the queue, causing all new jobs to use the
+        new configuration.</para>
+
+        <para><emphasis role="bold">maxJobs</emphasis>: controls the number of
+        jobs, specifically <emphasis>maxJobs</emphasis> sets the maximum
+        number of jobs.</para>
+
+        <para><emphasis role="bold">maxGraphs</emphasis>: limits the maximum
+        amount of graphs. It generally makes sense to keep this value below or
+        at the same number as <emphasis>maxJobs</emphasis>, since not all jobs
+        submit graphs and when they do the Thor jobs are not executing graphs
+        all the time. If there are more than 2 submitted (Thor) graphs, the
+        second would be blocked until the next Thor instance becomes
+        available.</para>
+
+        <para>The idea here is that jobs may spend significant amount of time
+        outside of graphs, such as waiting on a workflow state (outside of the
+        Thor engine itself), blocked on a persist, or updating super files,
+        etc. Then it makes sense for Thor to have a higher limit of concurrent
+        jobs (<emphasis>maxJobs</emphasis>) than graphs
+        (<emphasis>maxGraphs</emphasis> / Thor instances). Since Thor
+        instances (graphs) are relatively expensive (lots of pods/higher
+        resource use), while workflow pods (jobs) are comparatively
+        cheap.</para>
+
+        <para>Thus, the delivered (example) chart values defines
+        <emphasis>maxJobs</emphasis> to be greater than
+        <emphasis>maxGraphs</emphasis>. Jobs queued to a Thor aren't always
+        running graphs. Therefore it can make sense to have more of these
+        jobs, which are not consuming a large Thor and all its resources, but
+        restrict the max number of Thor instances running.</para>
+
+        <para>Thor has 3 components (that correspond to the resource
+        sections).</para>
+
+        <orderedlist>
+          <listitem>
+            <para>Workflow</para>
+          </listitem>
+
+          <listitem>
+            <para>Manager</para>
+          </listitem>
+
+          <listitem>
+            <para>Workers</para>
+          </listitem>
+        </orderedlist>
+
+        <para>The Manager and Workers are launched together and consume quite
+        a bit of resoures (and nodes) typically. While the Workflow is
+        inexpensive and usually doesn't require as many resources. You might
+        expect in a Kubernetes world, many of them would co-exist on the same
+        node (and therefore be inexpensive). So it makes sense for
+        <emphasis>maxJobs</emphasis> to be higher, and
+        <emphasis>maxGraphs</emphasis> to be lower</para>
```

Comment:
Yes.  I think it might be better to describe it the other way round even.  E.g.

  maxGraphs is the maximum number of thor clusters that you want to support.  However some thor jobs spend a significant quantity of their time outside of a graph - e.g. spraying a file, or processing super file operations.  So maxJobs can be set to a higher number to allow those low-resource jobs to run at the same time.